js/electron.js

```diff
@@ -47,7 +47,9 @@ function createWindow() {
 		electronOptionsDefaults.kiosk = true;
 	} else {
 		electronOptionsDefaults.fullscreen = true;
-		electronOptionsDefaults.autoHideMenuBar = true;
+		electronOptionsDefaults.frame = false;
+		electronOptionsDefaults.transparent = true;
+		electronOptionsDefaults.hasShadow = false;
```

Comment:
Marked this PR as draft.

I now added your `ready-to-show` event but I had to replace `mainWindow.maximize();` with `mainWindow.setFullScreen(true);`, otherwise the raspian os top bar was shown.

So I would be happy if you and other people can test this again ... @rejas @sdetweil @KristjanESPERANTO 

We need more than 2 tester for this, thanks.

Reply:
That's why `frame = false` was added; to prevent the top bar being shown. I will test with this approach as well.